.github/workflows/bazel.yml

```diff
@@ -0,0 +1,92 @@
+name: Build
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  macos_build:
+
+    runs-on: macos-11.0
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Write local.bazelrc File
+      shell: bash
+      run: |
+        cat >local.bazelrc <<EOF
+        common --config=ci
+        EOF
+
+    - name: Output the Bazel Info
+      shell: bash
+      run: |
+        bazelisk info
```

Comment:
I'm having trouble tracking down how you got bazelisk into here

Reply:
I am relying on bazelisk being installed in the runners.

[MacOS Software](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md)
[Ubuntu Software](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md)

Reply:
oh wow! Today I Learned